apps/judicial-system/digital-mailbox-api/src/app/modules/cases/dto/subpoena.dto.ts

```diff
@@ -0,0 +1,17 @@
+import { IsEnum, IsNotEmpty, IsOptional, IsString } from 'class-validator'
+
+import { ApiProperty } from '@nestjs/swagger'
+
+import { DefenderChoice } from '@island.is/judicial-system/types'
+
+export class UpdateSubpoenaDto {
+  @IsNotEmpty()
+  @IsEnum(DefenderChoice)
+  @ApiProperty({ enum: DefenderChoice })
+  defenderChoice!: DefenderChoice
+
+  @IsOptional()
+  @IsString()
+  @ApiProperty({ type: String })
+  defenderNationalId?: string
```

Comment:
Please consider making `defendantNationalId` required. What would be the result without a national id?

Reply:
We only need the defender national id if the defendant wants to choose a defender, if he chooses to waive his right, delay or delegate then there's no defender to assign

Reply:
Although perhaps I should add a check that makes sure its filled out if the defendant decides to choose a defender, so we throw an error if the defender is missing in that case

Reply:
Sorry, I confused this with the defendantNationalId